app/alert_screen.py

```diff
@@ -0,0 +1,247 @@
+                            id="fire_icon_rwo",
+                            children=fire_icon,
+                            style={
+                                "display": "flex",
+                                "justify-content": "center",
+                                "height": "30%",
+                            },
+                        ),
+                        dbc.Row(
+                            id="fire_text_row",
+                            children=fire_text_div,
+                            style={
+                                "display": "flex",
+                                "justify-content": "center",
+                                "margin-right": "2.5%",
+                            },
+                        ),
+                    ],
+                    style={
+                        "width": "50%",
+                        "margin": "2.5%",
+                    },
+                ),
+                dbc.Col(
+                    id="col_image_fire",
+                    children=[
+                        dbc.Row(
+                            html.Div(
+                                html.Div(
+                                    html.P("DÉPART DE FEU"),
+                                    style={
+                                        "font-size": "4vw",
+                                        "color": "#fd4848",
+                                        "font-weight": "bold",
+                                    },
+                                    className="blink-image",
+                                ),
+                            ),
+                            style={
+                                "display": "flex",
+                                "justify-content": "center",
+                                "padding-bottom": "7%",
+                            },
+                        ),
+                        dbc.Row(
+                            id="fire_images_row",
+                            children=fire_images_div,
+                            style={
+                                "display": "flex",
+                                "justify-content": "center",
+                                "margin-right": "2.5%",
+                            },
+                        ),
+                    ],
+                    style={
+                        "width": "50%",
+                        "margin": "2.5%",
+                    },
+                ),
+            ],
+            style={
+                "height": "100%",
+            },
+        ),
+    ]
+
+    style = {
+        "height": "100%",
+        "width": "100%",
+        "position": "fixed",
+    }
+
+    return layout_div, style
+
+
+# ----------------------------------------------------------------------------------------------------------------------
+# App layout
+# The following block gathers elements defined above and returns them via the alert_screen function
+def alert_screen():
```

Comment:
Maybe this one could be renamed into something like "AlertScreen" to keep the same naming convention as the one used for the "Homepage" function?

Reply:
It depends. `Homepage` does not follow PEP8 convention (i.e. use of snake case), this is upper Camel case. Do we want to keep it that way ? I don't really care to be honest

Reply:
I've changed it to `AlertScreen`

Reply:
Thanks a lot! I agree that it is slightly unorthodox but I feel like using the styling convention of a class reflects well the "instantiation" aspect of this function, which does not really encapsulate a logic but gathers previously defined function to return an object that did not exist before. Maybe I'm going too far ahah and anyway, it was not very important but many thanks for the change 😊